--- loupe-47.0/src/meson.build.orig	2024-09-13 13:40:16.000000000 +0200
+++ loupe-47.0/src/meson.build	2024-09-19 21:32:46.081999148 +0200
@@ -12,6 +12,7 @@ run_command(
 )
 
 cargo_options = ['--manifest-path', source_root / 'Cargo.toml']
+cargo_options += [ '--target', 'x86_64-unknown-linux-gnux32' ]
 cargo_options += ['--target-dir', build_root / 'src']
 
 if get_option('profile') == 'development'
@@ -53,7 +54,7 @@ custom_target(
   output: meson.project_name(),
   command: [
     'cp',
-    'src' / rust_target / meson.project_name(),
+    'src' / 'x86_64-unknown-linux-gnux32' / rust_target / meson.project_name(),
     '@OUTPUT@',
   ],
 )
